packages/client/src/views/Home/RecordAudit.tsx

```diff
@@ -0,0 +1,608 @@
+
+const getWQApplicationName = (application: GQLEventSearchSet): string => {
+  let name = ''
+  const applicationName =
+    (isBirthApplication(application) &&
+      application.childName &&
+      application.childName[0]) ||
+    (isDeathApplication(application) &&
+      application.deceasedName &&
+      application.deceasedName[0])
+
+  if (applicationName) {
+    if (applicationName.firstNames && applicationName.familyName) {
+      name = applicationName.firstNames + ' ' + applicationName.familyName
+    } else {
+      name = applicationName.familyName || applicationName.firstNames || ''
+    }
+  }
+  return name
+}
+
+const getGQLApplicationName = (application: IGQLApplication): string => {
+  let name = ''
+  const applicationName =
+    application && application.child && application.child.name
+      ? application.child.name[0]
+      : application.deceased &&
+        application.deceased.name &&
+        application.deceased.name[0]
+  if (applicationName) {
+    if (
+      (applicationName as GQLHumanName).firstNames &&
+      (applicationName as GQLHumanName).familyName
+    ) {
+      name =
+        (applicationName as GQLHumanName).firstNames +
+        ' ' +
+        (applicationName as GQLHumanName).familyName
+    } else {
+      name =
+        (applicationName as GQLHumanName).familyName ||
+        (applicationName as GQLHumanName).firstNames ||
+        ''
+    }
+  }
+  return name
+}
+
+const getLocation = (application: IApplication, props: IFullProps) => {
+  let locationType = ''
+  let locationId = ''
+  let locationDistrict = ''
+  let locationPermanent = ''
+  if (
+    application.event === 'death' &&
+    application.data &&
+    application.data.deathEvent
+  ) {
+    locationType =
+      (application.data.deathEvent.deathPlaceAddress &&
+        application.data.deathEvent.deathPlaceAddress.toString()) ||
+      ''
+    locationId =
+      (application.data.deathEvent.deathLocation &&
+        application.data.deathEvent.deathLocation.toString()) ||
+      ''
+    locationDistrict =
+      (application.data.deathEvent.district &&
+        application.data.deathEvent.district.toString()) ||
+      ''
+    locationPermanent =
+      (application.data.deceased &&
+        application.data.deceased.districtPermanent &&
+        application.data.deceased.districtPermanent.toString()) ||
+      ''
+  } else if (application.data) {
+    locationType =
+      (application.data.child &&
+        application.data.child.placeOfBirth.toString()) ||
+      ''
+    locationId =
+      (application.data.child &&
+        application.data.child.birthLocation &&
+        application.data.child.birthLocation.toString()) ||
+      ''
+    locationDistrict =
+      (application.data.child &&
+        application.data.child.district &&
+        application.data.child.district.toString()) ||
+      ''
+  }
+
+  if (locationType === 'HEALTH_FACILITY') {
+    const facility = get(props.resources.facilities, locationId) || ''
```

Comment:
Do you need to use `get` here or would `props.resources.facilities[locationId]` be enough?